configure.ac

```diff
@@ -226,13 +227,8 @@ AC_ARG_ENABLE([defaultflags],
               [enable_defaultflags=yes])
 AS_IF([test "x$enable_defaultflags" = "xyes"],
       [
-      ADD_PREPROC_FLAG([-D_DEFAULT_SOURCE])
-      ADD_PREPROC_FLAG([-D_BSD_SOURCE])
```

Comment:
`_BSD_SOURCE` was added in #716 to support old versions of glibc. Given that the PR author himself [suggested using `_GNU_SOURCE` instead](https://github.com/tpm2-software/tpm2-tss/pull/716#issuecomment-356776051), dropping it in favour of `AC_USE_SYSTEM_EXTENSIONS` should be fine.